IMS.BL/Inventory.cs

```diff
@@ -1,52 +1,44 @@
-using System.Collections.Generic;
-using System.Linq;
-using AutoMapper;
 using System;
+using System.Threading.Tasks;
+using IMS.BL.DataService;
 
 namespace IMS.BL
 {
     public class Inventory
```

Comment:
what is inventory ? a repo a service and if its one of them why not applying strategy pattern